wlauto/workloads/msword/__init__.py

```diff
@@ -79,12 +80,18 @@ class MsWord(AndroidUiAutoBenchmark):
                   The output is piped to log files which are then pulled from the phone.
                   '''),
         Parameter('use_test_file', kind=bool, default=False,
-                  description='If ``True`` then use a provided test file instead of creating one'),
-        Parameter('test_file', kind=str, description='Document to push to the device for testing'),
-        Parameter('test_image', kind=str, description='Image to be embedded in the document under test'),
```

Comment:
Isn't it "safer" to have this as a parameter so that file to be pushed can be explicitly specified, rather than assuming any jpg/jpeg file is valid?